src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Colors {
+    Yellow,
+    Green,
+    Black,
+    White,
+    Orange,
+    Puprle;
```

Comment:
Don’t forget how to name constants according to style guide (the same with naming of enum values)
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names